### PR TITLE
Feature | storage providers | add basespace sdk v2  | CROMSO-248

### PR DESCRIPTION
**Pull request by yairfischer**
Most files are generated by

swagger-codegen generate -i swagger-private-v2.json -l python -o .

### CODE REVIEW

File: project.mk
Comment (ikravets) [inline project.mk:1]: since you're removing emgbuild, you should probably remove this file as well

File: requirements.txt
Comment (ikravets) [inline requirements.txt:1]: why do you need `requirements.txt` when you're using `pyproject.toml`?

File: test-requirements.txt
Comment (ikravets) [inline test-requirements.txt:1]: see `requirements.txt` comment

File: git_push.sh
Comment (ikravets) [inline git_push.sh:1]: I don't think we need this file

File: setup.py
Comment (ikravets) [inline setup.py:1]: We probably don't need `setup.py` if we're using `pyproject.toml`

File: project.mk
Comment (a-lahat) [reply project.mk:1]: removed

File: requirements.txt
Comment (a-lahat) [reply requirements.txt:1]: removed

File: test-requirements.txt
Comment (a-lahat) [reply test-requirements.txt:1]: removed

File: git_push.sh
Comment (a-lahat) [reply git_push.sh:1]: removed

File: setup.py
Comment (a-lahat) [reply setup.py:1]: removed

**Review by ikravets — APPROVED** [review]

**Pull request merged**